runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/CredentialTile/CredentialTile.tsx

```diff
@@ -51,32 +52,27 @@ const CredentialTile: FC<CredentialTileProps> = ({
     };
 
     return (
-        <Tile leftBorderColor={`4px solid ${collectionColors[collection.color].hex}`} minHeight='min-content'>
-            <CredentialCard collectionColor={collectionColors[collection.color].hex} onClick={handleClick}>
+        <Tile leftborderbolor={`4px solid ${hexColor}`} minheight='min-content'>
```

Comment:
what is this naming convention called :D I'm guessing `lowercase`

Reply:
Yeah, this was the error in console, that the DOM element cannot have property which is not lowercased